docs/chainbridge/setup/outline.md

```diff
@@ -0,0 +1,20 @@
+---
+id: outline
+title: Outline
+---
+
+This guide walks through the setup for a bridge between a running Polygon PoS (Mumbai testnet) and a local Polygon SDK network.
+
+## Requirements
+
+In this guide, you will run PolygonSDK nodes, ChainBridge relayer, and cb-sol-cli tool which is a CLI tool to deploy contracts locally. The following environments are required before starting setup.
+
+* Go: >= 1.16
+* Node.js >= 16.13.0
+* Git
+
+In addition, you need to clone the following repositories with the versions to run some applications.
+
+* [Polygon SDK](https://github.com/0xPolygon/polygon-sdk.git): `6efebc6` on `develop` branch
+* [ChainBridge](https://github.com/ChainSafe/ChainBridge): `9585bc8` on `main` branch
```

Comment:
It would be great if we could retest and update the docs with a Release version instead of a Git commit: https://github.com/ChainSafe/ChainBridge/releases/tag/v1.1.5

@ZeljkoBenovic which version are you using?

Reply:
Fixed at 5756c760532fdfa7da42c47aad4dc893d1e4074e